tests/sles4sap/hana_test.pm

```diff
@@ -32,12 +36,18 @@ sub run {
     my $sid         = get_required_var('INSTANCE_SID');
     my $instance_id = get_required_var('INSTANCE_ID');
     my $sapadm      = $self->set_sap_info($sid, $instance_id);
+    my $password    = get_var('INSTANCE_PASSWORD', 'Qwerty_123');
+
+    # Test PIDs max, as SAP as some prerequisites on this
     $self->test_pids_max;
+
+    # We need to set the default password to the SAP one before swapping user
+    set_var('PASSWORD', $password);
```

Comment:
Is this needed in `sles4sap/tests/hana_test`?

Reply:
You just said above that `Both tests/sles4sap/hana_test and tests/sles4sap/hana_cluster rely on the PASSWORD variable`, so I think so :)

But I checked and I first though that it was needed for `user_change`  but seems not. In fact I'm not quite sure that setting `PASSWORD` is really needed. I will check in all files.

Reply:
> You just said above that `Both tests/sles4sap/hana_test and tests/sles4sap/hana_cluster rely on the PASSWORD variable`, so I think so :)

Hence my question. This is `hana_test` which uses the `PASSWORD` variable set in either `hana_install` or `wizard_hana_install`. IMO it would make sense to do a `set_var('PASSWORD', $password)` in the `hana_test` module only if it's needed by another module after, but I don't think we have such scenarios.